lib/library.rb

```diff
@@ -0,0 +1,29 @@
+class Library
+  attr_accessor :items
+
+  def initialize(options={})
+    @items = options[:items] || []
+  end
+
+  def check_out(user, args={})
+    if find_item(args)
+      check_out_item(user, @item)
+    else
+      binding.pry
+    end
+  end
+
+
+  private
+
+  def find_item(args={})
+    @item = @items.detect{|obj| obj[args.keys.first] == (args[args.keys.first]).to_s }
+  end
+
+  def check_out_item(user, item)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `item`. If it's necessary, use `_` or `_item` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)